tools/setup-windows/SetupWindows.cs

```diff
@@ -28,54 +39,68 @@ public static int Main (string [] args)
 			}
-			refAssembliesDirectories.Add (Path.Combine (progFiles, "Reference Assemblies", "Microsoft", "Framework", "MonoAndroid"));
-			
-			var msbuildTargets  = Path.Combine (vsInstall, "MSBuild", "Xamarin", "Android");
-			var newTargets      = Path.Combine (prefix, "lib", "xamarin.android", "xbuild", "Xamarin", "Android");
-			var newAssemblies   = Path.Combine (prefix, "lib", "xamarin.android", "xbuild-frameworks", "MonoAndroid");
+
+			links.Add (new SymbolicLink {
+				Source = Path.Combine (progFiles, "Reference Assemblies", "Microsoft", "Framework", "MonoAndroid"),
+				Target = newAssemblies,
+			});
+			links.Add (new SymbolicLink {
+				Source = Path.Combine (vsInstall, "MSBuild", "Xamarin", "Android"),
+				Target = Path.Combine (prefix, "lib", "xamarin.android", "xbuild", "Xamarin", "Android"),
+			});
+			links.Add (new SymbolicLink {
+				Source = Path.Combine (vsInstall, "MSBuild", "Xamarin", "Xamarin.Android.Sdk.props"),
+				Target = Path.Combine (prefix, "lib", "xamarin.android", "xbuild", "Xamarin", "Xamarin.Android.Sdk.props"),
+				IsFile = true,
+			});
+			links.Add (new SymbolicLink {
+				Source = Path.Combine (vsInstall, "MSBuild", "Xamarin", "Xamarin.Android.Sdk.targets"),
+				Target = Path.Combine (prefix, "lib", "xamarin.android", "xbuild", "Xamarin", "Xamarin.Android.Sdk.targets"),
+				IsFile = true,
+			});
 
 			if (Path.DirectorySeparatorChar != '\\') {
 				Console.Error.WriteLine ($"{AppName}: This program is for use on Windows.");
 				return 1;
 			}
 
 			if (args.Length == 0 || args.Any (v => string.Equals (v, "install", StringComparison.OrdinalIgnoreCase) || string.Equals (v, "/install", StringComparison.OrdinalIgnoreCase))) {
-				return Install (hash, msbuildTargets, newTargets, refAssembliesDirectories, newAssemblies);
+				return Install (hash, links);
 			}
 			if (args.Any (v => string.Equals (v, "uninstall", StringComparison.OrdinalIgnoreCase) || string.Equals (v, "/uninstall", StringComparison.OrdinalIgnoreCase))) {
-				var directories = new List<string> (refAssembliesDirectories);
-				directories.Add (msbuildTargets);
-				return Uninstall (hash, directories);
+				return Uninstall (hash, links);
 			}
 			Console.Error.WriteLine ($"{AppName}: Invalid command `{string.Join (" ", args)}`.");
 			return 1;
 		}
 
-		static int Install (string hash, string msbuildTargets, string newTargets, List<string> refAssembliesDirectories, string newAssemblies)
+		static int Install (string hash, List<SymbolicLink> links)
 		{
 			try {
-				foreach (var refAssemblies in refAssembliesDirectories) {
-					var backupAssemblies = GetNewBackupName (refAssemblies, hash);
-					Directory.CreateDirectory (Path.GetDirectoryName (refAssemblies));
-					if (!CreateSymbolicLink (refAssemblies, newAssemblies, backupAssemblies))
+				foreach (var link in links) {
+					var backup = GetNewBackupName (link.Source, hash);
+					if (!link.IsFile) {
+						Directory.CreateDirectory (Path.GetDirectoryName (link.Source));
```

Comment:
Wouldn't we *always* want to create the directory that `link.Source` goes into, even if -- especially if? -- it's a file?